frontend/pipeline.yml

```diff
@@ -289,3 +294,57 @@ publish:frontend:docker:
     expire_in: 1w
     paths:
       - .digests
+
+publish:frontend:licenses:
+  stage: publish
+  tags:
+    - hetzner-amd-beefy
+  rules:
+    - changes:
+        paths: ['frontend/**/*']
+        compare_to: '${RULES_CHANGES_COMPARE_TO_REF}'
+    - if: '$CI_COMMIT_REF_PROTECTED == "true"'
+      when: on_success
+  image: ${CI_DEPENDENCY_PROXY_DIRECT_GROUP_IMAGE_PREFIX}/denoland/deno:debian-2.0.2
+  needs:
+    - job: build:frontend:docker
+      artifacts: true
+  script:
+    - deno task --cwd frontend/scripts licenseFormatting --rootDir $(pwd)
+  artifacts:
+    when: on_success
+    expire_in: 1w
+    paths:
+      - licenses.md
+
+publish:frontend:licenses:docs-site:
```

Comment:
Maybe is it better to move this to the main `.gitlab-ci.yml` and use a single `publish:license:docs-site` for both backend and frontend?

Like (here)[https://github.com/mendersoftware/mender-server/pull/220/files#diff-037ea159eb0a7cb0ac23b851e66bee30fb838ee8d0d99fa331a1ba65283d37f7R431] setting the two jobs:

```
  needs:
    - job: publish:backend:licenses
      artifacts: true
    - job: publish:frontend:licenses
      artifacts: true
```
And get an unified `LICENSE.md` file, also for a single mender-docs page?

Reply:
Thanks!